src/Helpers.ts

```diff
@@ -0,0 +1,25 @@
+/** function latinCheck*/
+// no cyrillic letters allowed
```

Comment:
```suggestion
/**
 * Returns true if string doesn't contain cyrillic chars
 *
 * @param string - string to check
 */
```

Для остальных методов аналогично